frb_dart/lib/src/helpers.dart

```diff
@@ -89,3 +90,16 @@ mixin FlutterRustBridgeTimeoutMixin<T extends FlutterRustBridgeWireBase>
   @protected
   Duration? get timeLimitForExecuteNormal;
 }
+
+class MissingHeaderException implements Exception {
```

Comment:
uber-nit: Add `const MissingHeaderException();`. Then, when using it, no need to `throw MissingHeaderException()` but can `throw const MissingHeaderException()`